scalr/resource_scalr_workspace.go

```diff
@@ -484,6 +485,14 @@ func resourceScalrWorkspaceRead(d *schema.ResourceData, meta interface{}) error
 			"pre_apply":  workspace.Hooks.PreApply,
 			"post_apply": workspace.Hooks.PostApply,
 		})
+	} else if _, ok := d.GetOk("hooks"); ok {
```

Comment:
Also, I thought about fixing this problem by setting a default value for `hook` argument, but it is not supported for `TypeList` https://github.com/hashicorp/terraform-plugin-sdk/issues/142